projects/cyclegan_reference_simulations/README.md

```diff
@@ -127,6 +131,9 @@ This consists of:
 - Orographic forcing data: `gs://vcm-fv3config/data/orographic_data/v1.0`
 - Surface forcing data: `gs://vcm-fv3config/data/base_forcing/v1.1`
 
-Public versions of this data can be found
-[here](https://www.nco.ncep.noaa.gov/pmb/codes/nwprod/gfs.v16.3.7/fix/), and the
-specific files used can also be made available upon request.
+Public versions of the grid and orographic forcing data for a given resolution
+can be found
+[here](https://www.nco.ncep.noaa.gov/pmb/codes/nwprod/gfs.v16.3.7/fix/fix_fv3_gmted2010/),
+and public versions of the surface forcing data can be found
+[here](https://www.nco.ncep.noaa.gov/pmb/codes/nwprod/gfs.v16.3.7/fix/fix_am/).
+The specific files used can also be made available upon request.
```

Comment:
Nit: Instead of saying "available upon request" I'd just leave it at the gs:// links above, and maybe specify that the bucket is requester-pays.